runtime/ioext/ioext.go

```diff
@@ -0,0 +1,24 @@
+package ioext
```

Comment:
Package needs a doc comment... You can do it here, or create a `doc.go` file with this...